package.json

```diff
@@ -183,9 +183,9 @@
     "@types/webpack-hot-middleware": "2.25.9",
     "@types/yargs": "13.0.11",
     "@types/yargs-unparser": "2.0.1",
-    "@typescript-eslint/eslint-plugin": "5.62.0",
-    "@typescript-eslint/parser": "5.62.0",
-    "@typescript-eslint/utils": "5.62.0",
+    "@typescript-eslint/eslint-plugin": "6.21.0",
+    "@typescript-eslint/parser": "6.21.0",
+    "@typescript-eslint/utils": "6.21.0",
```

Comment:
we need to add `"@typescript-eslint/rule-tester": "6.21.0",` to root as well - single version policy for devDeps